Cargo.toml

```
@@ -21,7 +21,7 @@ open_url = ["webbrowser"]
 default_fonts = ["egui/default_fonts"]
 
 [dependencies]
-bevy = { version = "0.8", default-features = false, features = ["bevy_render", "bevy_core_pipeline", "bevy_asset"] }
+bevy = { git = "https://github.com/bevyengine/bevy", branch = "main", default-features = false, features = ["bevy_render", "bevy_core_pipeline", "bevy_asset"] }
```

```suggestion
bevy = { version = "0.9", default-features = false, features = ["bevy_render", "bevy_core_pipeline", "bevy_asset"] }
```

Bevy 0.9 is now released. :)